FirebaseCore/Sources/FIRLogger.m

```diff
@@ -76,10 +76,7 @@ void FIRLoggerInitializeASL(void) {
       [sFIRLoggerUserDefaults setBool:YES forKey:kFIRPersistedDebugModeKey];
       forceDebugMode = YES;
     }
-    GULLoggerInitializeASL();
-    if (overrideSTDERR) {
```

Comment:
This will need to be noted in the release note and might need a documentation update.

Reply:
Definitely agree about the release note since it was mentioned in [v4.6.0](https://firebase.google.com/support/release-notes/ios#version_460_-_november_14_2017). I did a search for `FIRLoggerForceSTDERR` and didn't see anything come up that wasn't in our codebase or a fork of it so I don't think there's _existing_ documentation to update at least.